trunk/src/protocol/srs_protocol_raw_avc.hpp

```diff
@@ -95,6 +95,13 @@ class SrsRawHEVCStream
     // @param flv output the muxed flv packet.
     // @param nb_flv output the muxed flv size.
     virtual srs_error_t mux_avc2flv(std::string video, int8_t frame_type, int8_t avc_packet_type, uint32_t dts, uint32_t pts, char **flv, int *nb_flv);
+    // Mux the hevc video packet to flv video packet, enhanced mode.
+    // @param packet_type, SrsVideoHEVCFrameTraitPacketTypeSequenceStart or SrsVideoHEVCFrameTraitPacketTypeCodedFrames.
+    // @param frame_type, SrsVideoAvcFrameTypeKeyFrame or SrsVideoAvcFrameTypeInterFrame.
+    // @param video the hevc raw data.
+    // @param flv output the muxed flv packet.
+    // @param nb_flv output the muxed flv size.
+    virtual srs_error_t mux_avc2flv_enhanced(std::string video, int8_t frame_type, int8_t packet_type, uint32_t dts, uint32_t pts, char **flv, int *nb_flv);
```

Comment:
is avc an alias for H.264? then is this method name can be renamed to:
 ```suggestion
    virtual srs_error_t mux_hevc2flv_enhanced(std::string video, int8_t frame_type, int8_t packet_type, uint32_t dts, uint32_t pts, char **flv, int *nb_flv);
```

Reply:
That's right, but there's no need to address it in this PR; please submit a separate one for that.

Reply:
Better to add a TODO comment for this.